filecoin-proofs/src/bin/paramfetch.rs

```diff
@@ -102,6 +102,19 @@ Defaults to '{}'
                 .long("ipget-bin")
```

Comment:
Blocking: This argument cannot be combined with `ipget-args` or `ipget-version`. Please do use `conflicts_with`.

Reply:
we should be able to combine this with `ipget-args`